hyp3lib/rtc2color.py

```diff
@@ -107,8 +136,12 @@ def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
   if real == True:
     red = (2.0*rp*(1 - blue_mask) + zp*blue_mask)
   else:
+    # FIXME: Should we *scale* this? There are values over 1
     red = (2.0*rp*(1 - blue_mask) + zp*blue_mask)*255
+  # FIXME: We should be *adding* 1 I think... values less than 1 will be between
+  #        the no data value and our minimum valid value...
   red[red==0] = 1
```

Comment:
It is my understanding that the `1` here is to allow for `0` to represent "no data" and data to range from 1--255.

However, we're allowing: 
* over saturation -- `(2.0*rp*(1 - blue_mask) + zp*blue_mask)` produces values above 1, so 255 will represent anything that as `>= 1`. Okay?
* We should probably be *adding* one so values that are between 0 and 1 are still counted as valid. Correct?

Reply:
Over-saturation is okay, and yes, we should be adding 1.